minclient/send.go

```diff
@@ -0,0 +1,170 @@
+// send.go - Send related routines.
+// Copyright (C) 2017  Yawning Angel.
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as
+// published by the Free Software Foundation, either version 3 of the
+// License, or (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public License
+// along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+package minclient
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/katzenpost/core/constants"
+	"github.com/katzenpost/core/crypto/rand"
+	"github.com/katzenpost/core/epochtime"
```

Comment:
To fetch correct epochtime, have to replace this with "github.com/hashcloak/Meson-client/pkiclient/epochtime".